### ISSUE TITLE
How do I Initialize Multiple Instances of MMDrawerController

### ISSUE THREAD

**Issue by robblock** [mutualmobile/MMDrawerController#436]
I am able to get one instance of MMDrawerController working, but I would like to have more side menus with different stylings.

Is this possible & How would I do this?

**Comment by ghost** [comment]
Are you referring to menus nested inside the initial menu?  Or a right side menu along with a left side menu?  Having another menu on the opposite side is as simple as repeating the same process for the first menu that you instantiated.

**Comment by joemlong** [comment]
Does MMDrawerController support menus nested inside the initial menu?  I would like to implement a hiearchial menu structure.

**Comment by ghost** [comment]
No, I don't believe it does.  At least I've never seen anything like that, and I'm not sure how it'd be implemented.
